zebra/debug.c

```diff
@@ -373,6 +374,15 @@ DEFPY (debug_zebra_neigh,
 	return CMD_SUCCESS;
 }
 
+DEFUN(debug_zebra_tc, debug_zebra_tc_cmd, "debug zebra tc",
```

Comment:
seems folded by git clang-format, maybe i should change it?

Reply:
yes ignore clang-format for these